go-selinux/label/label_stub.go

```diff
@@ -30,7 +30,7 @@ func Relabel(path string, fileLabel string, shared bool) error {
 // DisableSecOpt returns a security opt that can disable labeling
 // support for future container processes
 func DisableSecOpt() []string {
-	// TODO the selinux.DisableSecOpt stub returns []string{"disable"} instead of "nil"
+	// noling:godox // TODO the selinux.DisableSecOpt stub returns []string{"disable"} instead of "nil"
```

Comment:
Can you just remove the comment, It has been there for years and I am not sure it is correct.

Reply:
Done